Source/Rows/DateInlineRow.swift

```diff
@@ -105,7 +105,16 @@ open class _CountDownInlineRow: _DateInlineFieldRow {
 public final class DateInlineRow_<T>: _DateInlineRow, RowType, InlineRowType {
     required public init(tag: String?) {
         super.init(tag: tag)
-        onExpandInlineRow { cell, row, _ in
+        onExpandInlineRow { cell, row, inlineRow in
+            inlineRow.cellUpdate() { cell, row in
+                cell.datePicker.datePickerMode = .date
```

Comment:
I think setting datePickerMode is not needed here, right?

Reply:
From the documentation: default is UIDatePickerModeDateAndTime. So this line needs to stay for DateInlineRow and also the line that sets it to .time on the TimeInlineRow. But it looks like the line for DateTimeInlineRow that sets it to .dateAndTime can be removed.

Reply:
I mean `datePickerMode` is set in `DatePickerCell` (line 54). It is not necessary to re-set it here